ts/webui/src/components/Overview.tsx

```diff
@@ -167,7 +165,12 @@ class Overview extends React.Component<{}, OverviewState> {
                                         </div>
                                     </Stack>
                                     <div className='overviewChart'>
-                                        <Accuracy accuracyData={accuracyGraphData} accNodata={noDataMessage} />
+                                        <DefaultPoint
+                                            trialIds={bestTrials.map(trial => trial.info.trialJobId)}
+                                            visible={true}
```

Comment:
I think "visible" can be handled in this level. No need to pass a property into `DefaultPoint`.

Reply:
yes, here no need this `visible`. And also no need `whichGraph`. `whichGraph` is appeared in v0.8 for `ant design`. It should be deleted when using `fabric/fluent` because `current tab` will be uninstall when you choose other `tab`. So webui doesn't need to consider `current tab` when page refresh. Thanks for your comment.